docs/configuration.md

```diff
@@ -1705,6 +1705,26 @@ Apart from these, the following properties are also available, and may be useful
     <a href="job-scheduling.html#resource-allocation-policy">description</a>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.dynamicAllocation.recoverCachedData</code></td>
+  <td><code>false</code></td>
+  <td>
+  If dynamic allocation is enabled, and
```

Comment:
If you look at other blocks above, your indentation is off.

Reply:
fixed